README.md

```diff
@@ -26,28 +26,33 @@ This value can be overriden using `SCR_LOGGING_FORMAT` environment variable.
 Alternatively, an external `logback.xml` with much more customizations can be provided using `-Dlogback.configurationFile` JVM parameter.
 
 ## How to run service:
-TODO
-<!-- * Navigate to `ddocker/service`
+* Navigate to `docker/service`
 * Add environment tag `export TAG=latest`
 * Run script: `build-image.sh` (excute privileges might be required `chmod +x build-image.sh`)
 * Navigate to `docker`
 * Run script: `start-local-environment.sh`
 
 If gradle-wrapper.jar doesn't exist navigate to docker/service in terminal and run:
 * If gradle isn't installed `brew install gradle`
-* Update gradle `gradle wrapper` -->
+* Update gradle `gradle wrapper` 
 
 ## How to run unit tests:
-TODO
-<!-- * Navigate to `service`
-* Run: `./gradlew test` -->
+* Navigate to `service`
+* Run: `./gradlew test`
 
 ## How to run integration tests:
-TODO
-<!-- * Navigate to `service`
-* Run: `./gradlew integrationTest` -->
+* Navigate to `service`
+* Run: `./gradlew integrationTest`
+
+## How to run style check:
+* Navigate to `service`
+* Run: `./gradlew staticCodeAnalysis` 
 
 ## How to run all checks:
-TODO
-<!-- * Navigate to `service`
-* Run: `./gradlew check` -->
+* Navigate to `service`
+* Run: `./gradlew check` 
+
+### Licensing
+This code is dual licensed under the MIT license and the OGL (Open Government License). Any new work added to this repository must conform to the conditions of these licenses. In particular this means that this project may not depend on GPL-licensed or AGPL-licensed libraries, as these would violate the terms of those libraries' licenses.
+
+The contents of this repository are protected by Crown Copyright (C).
```

Comment:
should we have pre-requests, i.e. install docker, lombok plugin?

Reply:
I think what we have is fine for now.